src/stylesheets/elements/_table.scss

```diff
@@ -4,7 +4,7 @@ table {
   min-width: 100%;
 
   thead {
-    tr {
+    th {
```

Comment:
Why does this fix the issue (the `th` should already be nested in a a `tr`) and why was the rule `tr` before?